mantle/kola/tests/rhcos/upgrade.go

```diff
@@ -358,8 +360,25 @@ func downloadLatestReleasedRHCOS(target string) (string, error) {
 		return
 	}(releaseIndex, unique)
 
+	// In cases where there is a blocked upgrade to a new Y-stream there can be
+	// two nodes that don't have an edge to upgrade to. This is generally the
+	// latest 4.Y-1.Z and the latest 4.Y.Z. Choose the latest 4.Y.Z
+	if len(difference) < 1 {
+		return "", errors.New("Could not find the latest release")
+	}
+	latest := difference[0]
+	if len(difference) > 1 {
+		latestVersion := semver.New(graph.Nodes[latest].Version)
+		for _, v := range difference {
+			currentVersion := semver.New(graph.Nodes[v].Version)
+			if latestVersion.LessThan(*currentVersion) {
+				latest = v
```

Comment:
I think now we're missing

```suggestion
				latest = v
				latestVersion = currentVersion
```

Reply:
Whoops! Fixed